LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/Logger.cs

```diff
@@ -6,254 +6,108 @@ namespace LoRaWan
     using System;
     using System.Net;
     using System.Net.Sockets;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Devices.Client;
     using Microsoft.Extensions.Logging;
 
     public static class Logger
     {
-        // Interval where we try to estabilish connection to udp logger
-        private const int RETRY_UDP_LOG_CONNECTION_INTERVAL_IN_MS = 1000 * 10;
+        public static LogLevel LoggerLevel => sink.LogLevel;
 
-        public static LogLevel LoggerLevel => configuration.LogLevel;
+        private static ILogSink sink = new ConsoleLogSink(LogLevel.Error);
 
-        private static LoggerConfiguration configuration = new LoggerConfiguration();
-        private static volatile UdpClient udpClient;
-        private static IPEndPoint udpEndpoint;
-        private static volatile bool isInitializeUdpLoggerRunning;
-        private static Timer retryUdpLogInitializationTimer;
-
-        public static void Init(LoggerConfiguration loggerConfiguration)
+        public static void Init(LoggerConfiguration configuration,
```

Comment:
Yes, there will be a follow-up. This PR doesn't address #456 in its entirety otherwise it would have been very large. This PR prepares the ground work since `Logger` is now very thin and also implements more robust logging over TCP.